app/controllers/login_controller.rb

```diff
@@ -0,0 +1,6 @@
+class LoginController < ApplicationController
```

Comment:
I think we should create a template (`.erb`) for the LoginController with a input field for email address and a submit button. The email value will be passed to the backend controller on submission and be used to `sign_in_and_redirect`.

Additionally, try to see if you can only render this controller if an environment variable (ie. `LOCAL_LOGIN`) is set to true.